stacks-signer/src/signer.rs

```diff
@@ -1438,6 +1480,42 @@ impl Signer {
         Ok(())
     }
 
+    /// Send DKG results to the miner coordinator so that it can coordinate signing rounds
+    fn send_dkg_results(&mut self, dkg_public_key: &Point) {
+        let Some(dkg_results_bytes) = self.pending_dkg_results.remove(dkg_public_key) else {
+            warn!("{self}: No DKG results found for key: {dkg_public_key}");
+            return;
```

Comment:
Shouldn't this return an error?

Reply:
Yes it should. This code was basically just moved out as-is from `process_dkg`. I'll improve the error handling.

Reply:
Updated. I also changed the calling function to check for pending dkg results before calling this function to make the behavior more explicit and lead to fewer surprises.